src/test/java/net/sf/jabref/model/database/BibDatabaseTest.java

```diff
@@ -187,4 +188,27 @@ public void addSameStringIdTwiceThrowsKeyCollisionException() {
         database.addString(string);
         fail();
     }
+
+    @Test
+    public void testAddEntryEventReceivement() {
+        BibDatabase database = new BibDatabase();
+        BibEntry shouldBeEntry = new BibEntry();
+        TestEventListener tel = new TestEventListener();
+        database.registerListener(tel);
+        database.insertEntry(shouldBeEntry);
+        BibEntry isEntry = tel.getBibEntry();
+        assertEquals(shouldBeEntry, isEntry);
+    }
+
+    @Test
+    public void testRemoveEntryEventReceivement() {
+        BibDatabase database = new BibDatabase();
+        BibEntry shouldBeEntry = new BibEntry();
+        TestEventListener tel = new TestEventListener();
+        database.insertEntry(shouldBeEntry);
+        database.registerListener(tel);
+        database.removeEntry(shouldBeEntry);
+        BibEntry isEntry = tel.getBibEntry();
+        assertEquals(shouldBeEntry, isEntry);
```

Comment:
could you use `expected` and `actual` as prefixes instead of `shouldBe` and `is`? I know, that old JabRef code uses `shouldBe` and `asIs`, but this is not in line with JUnit.